integration/kubernetes/k8s-expose-ip.bats

```diff
@@ -44,7 +44,8 @@ setup() {
 	svcip=$(kubectl get service ${service} -o=json | jq '.spec.clusterIP' | sed 's/"//g')
 	svcport=$(kubectl get service ${service} -o=json | jq '.spec.ports[].port')
 	# And check we can curl the expected response from that IP address
-	curl http://$svcip:$svcport | grep "Hello, world!"
+	echo_msg="hello,world"
+	curl http://$svcip:$svcport/echo?msg=${echo_msg} | grep "$echo_msg"
```

Comment:
just a note - I wondered if a `grep -q` or similar would help us not get the `hello,world` line coming to stdout.. just a thought.